include/deal.II/non_matching/mapping_info.h

```diff
@@ -781,6 +797,16 @@ namespace NonMatching
 
 
 
+  template <int dim, int spacedim>
+  boost::signals2::connection
+  MappingInfo<dim, spacedim>::connect_is_reinitialized(
+    const std::function<void()> &set_is_reinitialized)
+  {
+    return is_reinitialized.connect(set_is_reinitialized);
+  }
```

Comment:
I wrote this function analogous to https://github.com/dealii/dealii/blob/03e6c36b5e30a2a0a65ad9d9a53f883c95bae063/include/deal.II/lac/qr.h#L563-L571 let me know what you think.